### PR TITLE
Added Affinidi to adopters

### PR DESCRIPTION
**Pull request by idestis**

### CODE REVIEW

**Comment by djamaile** [comment]
Welcome 👋 !!

We need the DCO sign off before we can merge it 😄

File: ADOPTERS.md
Comment (freben) [inline ADOPTERS.md:225]: Hey, it seems you forgot the last "i" in the company URL here.

**Comment by idestis** [comment]
> And the DCO again! :) Sorry for the trouble.

done

**Review by djamaile — DISMISSED** [review]

**Review by djamaile — CHANGES_REQUESTED** [review]
Seems like your local formatting tool brought some unnecessarily changes. Lets revert that 😄

**Comment by idestis** [comment]
> Seems like your local formatting tool brought some unnecessarily changes. Lets revert that 😄

My bad, sorry for this. Reverted and fixed only URL in my line with DCO.

File: ADOPTERS.md
Comment (djamaile) [inline ADOPTERS.md:226]: Almost there! You have a enter here which makes these last lines fall in their own table cell. So, if you do some backspaces then we should be fine 😄

**Review by djamaile — CHANGES_REQUESTED** [review]

**Comment by idestis** [comment]
@djamaile finally I did all changes :)

**Review by djamaile — APPROVED** [review]
Welcome, again :)

**Comment by idestis** [comment]
Now we have to wait for @freben to review additionally. I hope he will like it :)

**Review by freben — APPROVED** [review]
Not just like it, I love it!

**Pull request merged**
